robotbase/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -91,11 +88,9 @@ private void configureShuffleboard() {
     // }
 
     LoggerTab loggerTab = new LoggerTab();
-    ArrayList<ClosedLoopSubsystem> subsystems = new ArrayList<>();
-    // subsystems.add(m_shootSub);
-    // subsystems.add(m_intakeSub);
-    // subsystems.add(m_storageSub);
-    //subsystems.add(m_driveSub);
+    
+    // An array with all subsystems to be affected by failsafe. Add more as needed.
+    ClosedLoopSubsystem[] subsystems = {m_intakeSub, m_shootSub, m_climbSub, m_driveSub, m_visionSub};
```

Comment:
Might want to pass in an anonymous array here.

Reply:
Okay, will do.